packages/ember-metal/tests/utils_test.js

```diff
@@ -10,6 +10,7 @@ QUnit.module('Ember Metal Utils');
 QUnit.test('inspect outputs the toString() representation of Symbols', function() {
   // Symbol is not defined on pre-ES2015 runtimes, so this let's us safely test
   // for it's existence (where a simple `if (Symbol)` would ReferenceError)
+  /* jshint latedef: false */
   let Symbol = Symbol || null;
```

Comment:
This is pretty wild.  Can we just change it (future PR is fine)?

I think this is a better guard (below):

```js
if (typeof Symbol !== 'undefined') {

}
```

Then we don't have to do any global clobbering variables...